index.html

```diff
@@ -460,6 +460,76 @@ <h4>Definitions</h4>
       </section>
     </section>
 
+    <section>
+      <h2><dfn data-lt="created" data-dfn-type="dfn" id="created">created</dfn></h2>
+      <a href="https://www.w3.org/TR/did-core/#created">Created.</a>
```

Comment:
Hmm, just a single word link? Feels strange... can this be elaborated upon?

Reply:
For this I just copied the format that is already present. I definitely agree it needs to not be like this. But figured I'd make it consistent with what's there currently, then iterate on all of them!

Reply:
@rhiaro I agree with your approach, but properties like this need to be defined in JSON Schema as well to be added here.

Reply:
lol, nvm I see them... are these really allowed in did documents? I thought we had not resolved meta data in did documents discussion....

Reply:
Well, right now they're in the DID spec, and have been for quite some time. The metadata discussion is ongoing but afaik there's no PR to remove them yet. Ideally I'm hoping we can get the Registries as close to the spec as possible as fast as possible, then continue to update them as the spec changes, so this was just part of that aim.